docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

```diff
@@ -7,7 +7,7 @@ description: The 1Password Source provides a secure endpoint to receive Sign-in
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<img src={useBaseUrl('img/send-data/airtable-icon.png')} alt="airtable-icon.png" width="150" />
+<img src={useBaseUrl('img/send-data/1password-source-icon.png')} alt="1password-source-icon.png" width="150" />
```

Comment:
The image here is very low resolution. Instead, I'd utilize the high-res 1Password logo we use in other parts of the docs (path pasted below).

```suggestion
<img src={useBaseUrl('img/integrations/1password/1password.png')} alt="Thumbnail icon" width="75"/>
```